src/injected/visualization/tabbed-item.ts

```diff
@@ -8,4 +8,11 @@ export interface TabbedItem {
     focusIndicator?: FocusIndicator;
     tabOrder: number;
     shouldRedraw?: boolean;
+    itemType?: TabbedItemType;
+}
+
+export enum TabbedItemType {
+    RegularItem,
```

Comment:
I think we don't need the RegularItem: many tab stop instances (in the Assessment case) will be created without one such that specifying one only for this case doesn't help.